### PR TITLE
New strings for PR#2543

### PR DESCRIPTION
**Pull request by dannycolin**
Please do not merge yet. Waiting on https://github.com/mozilla/multi-account-containers/pull/2543 to get merged in the main repository.

### CODE REVIEW

**Comment by dannycolin** [comment]
They're used in [Manage Extension Shortcut](https://support.mozilla.org/en-US/kb/manage-extension-shortcuts-firefox).

**Comment by flodolo** [comment]
Thanks. I see `Container Shorcut` in the string, but `Contain Shortcut X` (with X being a number). Is that the same strings concatenated to a number? 🤔

**Comment by dannycolin** [comment]
>  Is that the same string concatenated to a number?

Yes. AFAIK, it isn't possible to have a string with a placeholder for strings we want to translate in manifest.json.

**Comment by flodolo** [comment]
Can these strings be moved out of manifest.json in any way? That's a poor approach for localization.

**Comment by dannycolin** [comment]
> Can these strings be moved out of manifest.json in any way? That's a poor approach for localization.

I haven't tested but it might be possible to use a combinaison of [browser.runtime.onStartup()](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/onStartup) and [browser.commands.update()](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/commands/update). It looks a bit like a hackaround to me but it might be an acceptable compromise.

File: en/messages.json [hunk 1]
```diff
@@ -399,5 +399,18 @@
...
     "message": "Default Container",
     "description": "The title of the default container"
+  },
+  "openContainerPanel": {
+    "message": "Open container panel",
+    "description": "Option in the keyboard shortcut manager"
+  },
+  "containerShortcut": {
+    "message": "Container shortcut $keyId$",
+    "description": "Option in the keyboard shortcut manager",
```
Comment (flodolo) [inline en/messages.json:409]:
```suggestion
    "description": "Label for option in the keyboard shortcut manager. $keyId$ is a number from 1 to 10.",
```

File: en/messages.json [hunk 2]
```diff
@@ -399,5 +399,18 @@
   "defaultContainerLabel": {
     "message": "Default Container",
     "description": "The title of the default container"
+  },
+  "openContainerPanel": {
+    "message": "Open container panel",
+    "description": "Option in the keyboard shortcut manager"
```
Comment (flodolo) [inline en/messages.json:405]:
```suggestion
    "description": "Label for option in the keyboard shortcut manager (associating a shortcut to open the container panel)"
```

**Review by flodolo — APPROVED** [review]
Couple of nits on the comment, but strings look good. Thanks for removing the hard-coded bits.

File: en/messages.json (hunk 1 above)
Comment (dannycolin) [reply en/messages.json]: Done.

**Comment by dannycolin** [comment]
On [Luke's suggestion](https://github.com/mozilla/multi-account-containers/pull/2543#discussion_r1237244834), @flodolo can you merge this PR before we merge [multi-account-containers#2543](https://github.com/mozilla/multi-account-containers/pull/2543) please?

**Pull request merged**